easy-ui-react/package.json

```diff
@@ -0,0 +1,4 @@
+{
+  "name": "@easypost/easy-ui",
```

Comment:
how do we feel about this being the primary package name for our React components, and more generally as the prefix for other packages in this repo?

i.e.:

```ts
import { Button } from "@easypost/easy-ui";
```

other thoughts:

```ts
import { Button } from "@easypost/ui";
import { Button } from "@easypost/ezui";

import { DownloadIcon } from "@easypost/easy-ui-icons";
import { DownloadIcon } from "@easypost/ezui-icons";
import { DownloadIcon } from "@easypost/ui-icons";

import { colors } from "@easypost/easy-ui-tokens";
import { colors } from "@easypost/ezui-tokens";
import { colors } from "@easypost/ui-tokens";
```